.github/workflows/deploy.yml

```diff
@@ -125,46 +144,101 @@ jobs:
 
       - name: Delete clock to force its recreation
         run: |
-          cf delete -f "apply-clock-${PAAS_APP_ENVIRONMENT}"
+          make ${{ matrix.environment }} ci delete-clock
         env:
-          CONFIRM_PRODUCTION: true
+          PR_NUMBER:                ${{ env.PR_NUMBER }}
 
       - name: Terraform init, plan & apply
         run: make ${{ matrix.environment }} ci deploy
         env:
-          ARM_ACCESS_KEY:           ${{ secrets.TERRAFORM_STATE_ACCESS_KEY }}
+          ARM_ACCESS_KEY:           ${{ secrets.ARM_ACCESS_KEY }}
           IMAGE_TAG:                ${{ github.event.inputs.sha }}
+          PR_NUMBER:                ${{ env.PR_NUMBER }}
           TF_VAR_azure_credentials: ${{ secrets.AZURE_CREDENTIALS }}
           CONFIRM_PRODUCTION: true
 
+      - name: Check if a review environment exists
+        id: review-environment-exists
+        if:  github.event.inputs.pr != ''
+        uses: actions/github-script@v5
+        with:
+          result-encoding: string
+          script: |
+
+            let environmentName = process.env.PR_NUMBER;
+
+            github.rest.repos.getEnvironment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                environment_name: environmentName
+            }).then(res => {
+
+                console.log(`The environment ${environmentName} exists in the repository.`)
+                return "true"
+
+            }).catch(err => {
+
+                if (err.status == "404") {
+                    console.log(`The environment ${environmentName} does not exist in the repository.`)
+                    return "false"
+                }
+
+                console.error(err)
+
+            });
+
+      - name: Seed Review App
+        if:  github.event.inputs.pr != '' && steps.review-environment-exists.outputs.result == 'false'
+        run: |
+          cf ssh apply-pr-${PR_NUMBER} -c "export DISABLE_DATABASE_ENVIRONMENT_CHECK=1 && cd /app && /usr/local/bin/bundle exec rake db:schema:load setup_local_dev_data setup_all_provider_relationships" || true
```

Comment:
discussed in person: we could potentially get rid of schema load and just keep the seed file because `db:migrate` runs on every container startup

Reply:
I've removed schema load in a separate commit, we can give it a shot as it makes sense.